src/camps/tests.py

```diff
@@ -0,0 +1,14 @@
+import datetime
+
+from django.test import Client, TestCase
+from django.urls import reverse
+
+
+class CampMenuTest(TestCase):
+    def test_this_year_shown_on_homepage(self):
+        """By March, the current year's camp should be on the homepage."""
+        c = Client()
```

Comment:
Nitpick: there is already a client at `self.client`.
Also variables should be meaningful and not just single letters or abbreviations :)